tests/unit/TodoList1.spec.ts

```diff
@@ -2321,52 +2321,115 @@ describe('TodoList - Click Tests - IA Corrections', () => {
-    sessionStorage.setItem('EDIT_URL', 'https://edit.url/')
-    sessionStorage.setItem('BUSINESS_ID', 'BC1234567')
-    sessionStorage.setItem('CURRENT_ACCOUNT', '{ "id": "2288" }')
-    store.state.identifier = 'BC1234567'
-    // init draft Correction filing task
-    store.state.tasks = [
-      {
-        task: {
-          filing: {
-            header: {
-              name: 'correction',
-              status: 'DRAFT',
-              filingId: 123,
-              comments: []
-            },
-            business: {},
-            correction: {
-              correctedFilingType: 'incorporationApplication'
+  const editTests = [
+    { businessId: 'BC1234567', correctedFilingType: 'alteration' },
+    { businessId: 'BC1234567', correctedFilingType: 'incorporationApplication' },
+    { businessId: 'FM1234567', correctedFilingType: 'changeOfRegistration' },
+    { businessId: 'BC1234567', correctedFilingType: 'correction' },
+    { businessId: 'FM1234567', correctedFilingType: 'registration' }
+  ]
+
+  for (const test of editTests) {
+    it(`redirects to Edit URL to resume a draft ${test.correctedFilingType} correction`, async () => {
+      // init store
+      sessionStorage.clear()
+      sessionStorage.setItem('EDIT_URL', 'https://edit.url/')
+      sessionStorage.setItem('BUSINESS_ID', test.businessId)
+      sessionStorage.setItem('CURRENT_ACCOUNT', '{ "id": "2288" }')
+      store.state.identifier = test.businessId
+      // init draft Correction filing task
+      store.state.tasks = [
+        {
+          task: {
+            filing: {
+              header: {
+                name: 'correction',
+                status: 'DRAFT',
+                filingId: 123,
+                comments: []
+              },
+              business: {},
+              correction: {
+                correctedFilingType: test.correctedFilingType
+              }
             }
-          }
-        },
-        enabled: true,
-        order: 1
-      }
-    ]
-    store.state.keycloakRoles = ['staff'] // only staff may resume draft corrections
+          },
+          enabled: true,
+          order: 1
+        }
+      ]
+      store.state.keycloakRoles = ['staff'] // only staff may resume draft corrections
+
+      const wrapper = mount(TodoList, { store, vuetify })
+      const vm = wrapper.vm as any
+      await Vue.nextTick()
+
+      expect(vm.todoItems.length).toEqual(1)
+      expect(wrapper.find('.todo-subtitle span').text()).toBe('DRAFT')
+
+      await wrapper.find('.btn-draft-resume').trigger('click')
+
+      // verify redirection
+      const accountId = JSON.parse(sessionStorage.getItem('CURRENT_ACCOUNT'))?.id
+      const editUrl = `https://edit.url/${test.businessId}/correction/?correction-id=123`
+      expect(window.location.assign).toHaveBeenCalledWith(editUrl + '&accountid=' + accountId)
+
+      wrapper.destroy()
+    })
+  }
+
+  const localTest = ['annualReport', 'changeOfAddress', 'changeOfDirectors', 'conversion']
```

Comment:
Added tests to check all the correction types that stay in the filings ui

Reply:
Do the reviewers think that there is any value added to testing the default redirection?

Reply:
I think so. That way, a test breaks if we do something different with these filing types.